src/app/store/game-settings.store.ts

```diff
@@ -11,16 +12,29 @@ export class GameSettingsStore extends Store<GameSettingsState> {
     super(createInitialState());
   }
 
+  private getPlayerIdentifier(newPlayerName?: string, playerList?: Player[]) {
+    let playerIdentifier = (newPlayerName?.length ?? 0) > 0
+                            ? newPlayerName![0].toLocaleUpperCase()
+                            : '';
```

Comment:
can two players have string empty as identifier?

Reply:
Practically, you can't add the player if it has an empty identifier. So the user need to write a name which results in a possible identifier